Sources/SwiftParser/Attributes.swift

```diff
@@ -190,7 +197,14 @@ extension Parser {
       shouldParseArgument = false
     }
     if shouldParseArgument {
-      let (unexpectedBeforeLeftParen, leftParen) = self.expect(.leftParen)
+      var (unexpectedBeforeLeftParen, leftParen) = self.expect(.leftParen)
+      if attributeName.raw.trailingTriviaByteLength > 0 || leftParen.leadingTriviaByteLength > 0 {
```

Comment:
(The previous comment was for this part)

Does it make sene to add `unexpectedBeforeLeftParen  == nil` here? For `attrName<unexpected> (` .... Diagnosing trivia between `attrName` and `(` doesn't make much sense.

Reply:
Yes, good idea 👍🏽